ios/RNZoomUs.m

```diff
@@ -239,4 +243,24 @@ - (void)onMeetingError:(MobileRTCMeetError)errorCode message:(NSString *)message
   meetingPromiseReject = nil;
 }
 
+- (void)onClickShareScreen:(UIViewController *)parentVC {
+  if (@available(iOS 12.0, *)) {
```

Comment:
This section prefills the iOS SystemBroadcastPicker (accessible from the button in the Control Center) with the extension the developer has implemented as part of the same xcode project.